themes/base.theme.bash

```diff
@@ -106,7 +106,7 @@ function _bash_it_appearance_scm_init() {
 		fi
 	fi
 }
-_bash_it_appearance_scm_init
+_bash_it_library_finalize_hook+=('_bash_it_appearance_scm_init')
```

Comment:
I think we should add a documentation about this change, as it will be hard to notice this feature otherwise

Reply:
Yes! I don't have any brain worms going about the documentation. Can you suggest where and I'll just add something to start?

Reply:
Probably in `development.rst`, near the `Plugin Disable Callbacks` section

Reply:
This file format is wild. Any reason we don't just use markdown? GitHub natively supports markdown, like, everywhere and it doesn't need special tools to build or anything.

Reply:
rst vs markdown is a long debate. I like rst because I like sphinx and [readthedocs](https://bash-it.readthedocs.io/en/latest/). We can discuss it more if you would like 😄